Nos/Views/NotificationCard.swift

```diff
@@ -44,6 +44,10 @@ struct NotificationCard: View {
                             .font(.body)
                             .foregroundColor(.primaryTxt)
                             .tint(.accent)
+                            .environment(\.openURL, OpenURLAction { url in
```

Comment:
We have this same modifier repeated many times in the code. Maybe its time to refactor it to a ViewModifier we can reuse.

Reply:
Good idea! Done.